src/olympia/versions/tests/test_views.py

```diff
@@ -522,6 +522,7 @@ def test_download_for_unlisted_addon_reviewer(self):
     def test_download_for_admin(self):
         """File downloading is allowed for admins."""
         self.grant_permission(self.user, 'Reviews:Admin')
+        self.addon.authors.clear()
```

Comment:
That doesn't look like it's testing the actual changes. I think there are some tests missing?

Reply:
It is actually testing the changes : that test covers every case (disabled, deleted, etc) for admins. The problem was, before I added this single line, the user in the test was also an author of the add-on, and thus the test wasn't testing what it should have been testing...

Reply:
Oh, I missed that. Thanks for the explanation!